src/index.html

```diff
@@ -9,10 +9,45 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/index.css"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="body">
+    <h1 hidden>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="./images/imac.jpeg"
+        alt="imac"
+        class="card__image"
+      />
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina
+        <br />
```

Comment:
Avoid using <br> to create line breaks between inline elements or text. Instead, consider using CSS to manage the layout of elements. For instance, you can use a block-level element or control the width of the element to naturally wrap text.